src/App.jsx

```diff
@@ -14,33 +16,100 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+// export const handleLengthSort = () => {
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+// };
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+const SORT_ALPHABET = 'alphabetic';
+const SORT_LENGTH = 'length';
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+export const sortGoods = (goods, params) => {
+  const { sortBy, isReversed } = params;
+  const sortedGoods = [...goods];
+
+  if (sortBy.length) {
```

Comment:
The condition `if (sortBy.length)` will always evaluate to true since `sortBy` is a string. Consider changing this to `if (sortBy)` to check if `sortBy` is not an empty string.

Reply:
> The condition `if (sortBy.length)` will always evaluate to true since `sortBy` is a string. Consider changing this to `if (sortBy)` to check if `sortBy` is not an empty string.

Good point